Standards/scs-XXXX-vN-taxonomy-of-failsafe-levels.md

```diff
@@ -0,0 +1,254 @@
+| Ephemeral VM       | Equals the `server` resource in Nova, booting from ephemeral storage.                                                                    | L1, L2, L3, L4    |
+| Volume-based VM    | Equals the `server` resource in Nova, booting from a volume.                                                                             | L2, L3, L4        |
+| Ephemeral Storage  | Disk storage directly supplied to a virtual machine by Nova. Different from volumes.                                                     | L1, L2, L3, L4    |
+| Ironic Machine     | A physical host managed by Ironic or as a `server` resource in Nova.                                                                     | L1, L2, L3, L4    |
+| (Glance) Image     | IaaS resource usually storing raw disk data. Managed by the Glance service.                                                              | (L1), L2, L3, L4  |
+| (Cinder) Volume    | IaaS resource representing block storage disk that can be attached as a virtual disk to virtual machines. Managed by the Cinder service. | (L1, L2), L3, L4  |
+| (Volume) Snapshot  | Thinly-provisioned copy-on-write snapshots of volumes. Stored in the same Cinder storage backend as volumes.                             | (L1, L2), L3, L4  |
+| Volume Type        | Attribute of volumes determining storage details of a volume such as backend location or whether the volume will be encrypted.           | L3, L4            |
+| (Barbican) Secret  | IaaS resource storing cryptographic assets such as encryption keys. Managed by the Barbican service.                                     | L3, L4            |
+| Key Encryption Key | IaaS resource, used to encrypt other keys to be able to store them encrypted in a database.                                              | L3, L4            |
+| Floating IP        | IaaS resource, an IP that is usually routed and accessible from external networks.                                                       | L3, L4            |
+
+#### KaaS Layer (Kubernetes Resources)
+
+A detailed list of consequnces for certain failures can be found in the [Kubernetes docs](https://kubernetes.io/docs/tasks/debug/debug-cluster/).
+The following table gives an overview about certain resources on the KaaS Layer and in which failsafe classes they are affected:
+
+| Resource(s)        | Explanation                                                                                                                              | Affected by Level |
+| ------------------ | ---------------------------------------------------------------------------------------------------------------------------------------- | ----------------- |
+| Pod                | Kubernetes object that represents a workload to be executed, consisting of one or more containers.                                       | ???               |
```

Comment:
I want to ask if the `Affected by Level` column will be filled in, or are the question marks intended to remain there?

Reply:
It should be filled. But that is the point I need help with, as I have up to now mostly worked with the IaaS Layer, not the KaaS Layer.

Reply:
Hi @michal-gubricky and @josephineSei,
@DEiselt and I removed the questions marks now to fill in the levels

Reply:
Thanks, I will take a look at it.

Reply:
The changes look good to me. I will resolve this thread.